tests/migration_test.go

```diff
@@ -3366,6 +3361,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 
 					// Drain node.
 					node := vmi.Status.NodeName
+					libnode.Taint(node, libnode.GetNodeDrainKey(), k8sv1.TaintEffectNoSchedule)
```

Comment:
Might it be good please to make a function on a follow PR that does those two operations (this and drainNode) ?
as they appear more times and should come together, this way same problems can we avoided on the future

Reply:
Makes sense, but I'd rather have this as one of the requirements for https://github.com/kubevirt/kubevirt/pull/7188 .